file_picker/__init__.py

```diff
@@ -1,29 +1,9 @@
+# Aliases allow users to import directly from file_picker.*
+# FIXME: Should we deprecate this behavior? setup.py can't access __version__ unless django and deps are installed
```

Comment:
I guess we could wrap these imports in try/except ImportError: pass.  With a comment explaining the reason.

But I'd be happy deprecating this too.